### PR TITLE
New flag to show a breakdown of tokens used

### PR DESCRIPTION
**Pull request by Intron014**
To see a breakdown of where the tokens go, whether it is a prompt token or a completion token and therefore optimize the prompts if necesary!

### CODE REVIEW

**Comment by reorx** [comment]
Great suggestion. I was thinking that maybe we should move other figures concerning tokens in the `stat:` section to this new `token:` section. What is your opinion?

**Comment by Intron014** [comment]
I thought that too, but then we would leave the stats section without much. Maybe moving the token price and leaving total tokens?

**Comment by reorx** [comment]
My suggestion is:

```python
        if Config.verbose:
            print(blue(f'stat: sent_messages={len(messages)} total_messages={len(self.messages)}  price=~${"{:.6f}".format(data["usage"]["total_tokens"]/1000*0.002)}'))
        if Config.tokens:
            print(blue(f'tokens: prompt_tokens={data["usage"]["prompt_tokens"]} completion_tokens={data["usage"]["completion_tokens"]} total_tokens={data["usage"]["total_tokens"]}'))
```
- move total_tokens to `tokens:` section
- rename tokens_price to price
- keep colors the same for a clearer and more consistent UI

**Comment by Intron014** [comment]
Yeah, actually, that looks pretty good!!

File: ai.py [hunk 1]
```diff
@@ -244,10 +244,10 @@ def __init__(self, api_base_url, api_key, conversation=False, messages=None):
     def chat(self, user_message, params=None):
         self.messages.append(user_message)
         res_message, data, messages = self.create_completion(params=params)
-        if Config.verbose:
-            print(blue(f'stat: sent_messages={len(messages)} total_messages={len(self.messages)} total_tokens={data["usage"]["total_tokens"]} tokens_price=~${"{:.6f}".format(data["usage"]["total_tokens"]/1000*0.002)}'))
+                if Config.verbose:
```
Comment (reorx) [inline ai.py]: I believe this is an accident

Comment (Intron014) [reply ai.py]: Very true! Hahah

**Pull request merged**